lib/compress/zstd_compress_internal.h

```diff
@@ -331,6 +332,9 @@ struct ZSTD_CCtx_params_s {
 
     /* Internal use, for createCCtxParams() and freeCCtxParams() only */
     ZSTD_customMem customMem;
+
+    /* Controls prefetching in some dictMatchState matchfinders */
+    ZSTD_paramSwitch_e prefetchCDictTables;
```

Comment:
One more thing: add to this comment that the behavior is controlled from the cctx ms, and that this parameter has no effect in the cdict ms.